### ISSUE TITLE
Is it possible to execute code in markdown?

### ISSUE THREAD

**Issue by AbstProcDo** [typora/typora-issues#4005]
Is it possible to run code in markdown as it running in emacs org-babel?

For instance:

```python
print(3)
```
execute it and get result?

**Comment by JulianWgs** [comment]
No, use [Jupyter ](https://jupyter.org/) instead. Jupyter lab has markdown editor and can execute Python with rich outputs.

**Comment by CyrusYip** [comment]
You can run code in Rmarkdown.
